029-IoTEdge/README.md

```diff
@@ -0,0 +1,74 @@
+
+This hack includes presentations that feature lectures introducing key topics associated with each challenge. It is recommended that the host present each lecture before attendees kick off that challenge.
+
+## Learning Objectives
+
+
+In this hack you will solve common challenges for companies planning to use Azure IoT in Industrial IoT scenarios. This includes:
+
+1. Deploying and configuring the IoT Edge runtime
+1. Deploying modules to the running IoT Edge device
+1. Managing/monitoring/configuring IoT Edge devices at scale from IoT Hub
+1. Configuring an OPC (simulator) to generate factory data
+1. Interfacing IoT Edge to the factory simulator to capture data and push to the cloud
+1. Consuming data published to the cloud for analysis and reporting
+
+## Challenges
+
+- Challenge 1: **[Deploy IoTHub/Edge](Student/Challenge-01.md)**
+  - Get familiar with the basic concepts of Azure IoT Hub and Azure IoT Edge.
+  - Iot Hub Creation
+  - Edge Device creation
+
+- Challenge 2: **[Deploy OPC Simulator](Student/Challenge-02.md)** 
+  - OPC Simulator to serve as our virtual 'factory'
+
+- Challenge 3: **[Deploy Industrial IoT solution](Student/Challenge-03.md)**
+  - Deploy Industrial IoT platform to the IoT Edge
+
+- Challenge 4: **[Route messages and do time-series analysis](Student/Challenge-04.md)**
+    - Route IoT Hub data to Event Hub
+    - Route Event Hub to a time series data store
+
+- Challenge 5: **[Process Steaming Data](Student/Challenge-05.md)**
+  - Stream processing
+    - Reading from IoT Hub
+    - Aggregating/filtering data (querying)
+    - Output data to data lake & Power BI
+
+- Challenge 6: **[Deploy to devices at scale](Student/Challenge-06.md)**
+  - Use Deployment manifests to deploy modules to IoT Edge devices at scale
+
+- Challenge 7: **[Connect to Devices with Device Streams](Student/Challenge-07.md)**
+  - Use Azure IoT Hub device streams (in preview) to connect to IoT Devices over SSH
+  - Close down SSH conectivity in a Firewall to confirm remote access is not impacted
+
+
+## Repository Contents
+
+- `../Coach/Presentations`
+  - Contains all presentations listed in the Introduction above.
+- `../Coach/Assets`
```

Comment:
Your folder name is 'assets', need to make the A lowercase. However the Coach guide is correct.